<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>836</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bt_startDraw">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始绘制</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_K">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_K">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>渗透率       K：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_K_u">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(10^-3um^2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hp_u">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_hp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>气井打开厚度hp：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rg_u">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(f)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_rg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>气体相对密度rg：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rw_u">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_u">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(mPa.s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r1_u">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rw">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_r1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_u">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>气体黏度     u：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>井半径      rw：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>半球面向心流半径  r1：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S_u">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(f)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>表皮系数           S：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_re">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_re_u">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_re">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>供给半径          re：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_q">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_q">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>气井射孔段底部产量 q：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_q_u">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(10^4m^3/d)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_h2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fi">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fi">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>孔隙度           ψ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_h2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>气井射孔段底部高度h2：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fi_u">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(f)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_h2_u">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Swi_u">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(f)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sgr">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>160</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>残余气饱和度    Sgr：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sgr_u">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(f)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Swi">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Swi">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>束缚水饱和度    Swi：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Sgr">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wg_torusDrawWindow" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 217, 217);</string>
    </property>
   </widget>
   <widget class="QWidget" name="wg_plotDrawWindow" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>481</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_clearInput">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_clearDraw">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除绘制</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>底水气藏水锥动态模拟软件</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_resultView">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_savePlotDraw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_saveTorusDraw">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>740</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_u">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(a)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_drawBreakthroughTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>水锥突破曲线绘制</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_drawInputTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入时间曲线绘制：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_copyData">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>740</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>复制数据</string>
    </property>
   </widget>
   <zorder>bt_saveTorusDraw</zorder>
   <zorder>bt_startDraw</zorder>
   <zorder>lineEdit_K</zorder>
   <zorder>label_K</zorder>
   <zorder>label_K_u</zorder>
   <zorder>label_hp_u</zorder>
   <zorder>lineEdit_hp</zorder>
   <zorder>label_hp</zorder>
   <zorder>label_rg_u</zorder>
   <zorder>lineEdit_rg</zorder>
   <zorder>label_rg</zorder>
   <zorder>label_rw_u</zorder>
   <zorder>label_u_u</zorder>
   <zorder>label_r1_u</zorder>
   <zorder>lineEdit_u</zorder>
   <zorder>lineEdit_rw</zorder>
   <zorder>lineEdit_r1</zorder>
   <zorder>label_u</zorder>
   <zorder>label_rw</zorder>
   <zorder>label_r1</zorder>
   <zorder>label_S_u</zorder>
   <zorder>label_S</zorder>
   <zorder>lineEdit_S</zorder>
   <zorder>lineEdit_re</zorder>
   <zorder>label_re_u</zorder>
   <zorder>label_re</zorder>
   <zorder>lineEdit_q</zorder>
   <zorder>label_q</zorder>
   <zorder>label_q_u</zorder>
   <zorder>lineEdit_h2</zorder>
   <zorder>lineEdit_fi</zorder>
   <zorder>label_fi</zorder>
   <zorder>label_h2</zorder>
   <zorder>label_fi_u</zorder>
   <zorder>label_h2_u</zorder>
   <zorder>label_Swi_u</zorder>
   <zorder>label_Sgr</zorder>
   <zorder>label_Sgr_u</zorder>
   <zorder>lineEdit_Swi</zorder>
   <zorder>label_Swi</zorder>
   <zorder>lineEdit_Sgr</zorder>
   <zorder>wg_torusDrawWindow</zorder>
   <zorder>wg_plotDrawWindow</zorder>
   <zorder>bt_clearInput</zorder>
   <zorder>bt_clearDraw</zorder>
   <zorder>label_title</zorder>
   <zorder>tb_resultView</zorder>
   <zorder>bt_savePlotDraw</zorder>
   <zorder>lineEdit_time</zorder>
   <zorder>label_time_u</zorder>
   <zorder>rb_drawBreakthroughTime</zorder>
   <zorder>rb_drawInputTime</zorder>
   <zorder>bt_copyData</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action_sens"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_sens">
   <property name="text">
    <string>敏感性分析</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
